DedicatedServer/Scripts/Libs/Zrx/ModeLibs/FlagRush/UI/Modules/Header.Script.txt

```diff
@@ -156,6 +156,11 @@ Text GetManialink() {
 				QuadCenterFlag.Visible = !IsTimeLeft;	
 				QuadCenterFlag.Colorize = GetTeamPrimaryColor(FlagCarrierTeam);
 
+				/* Low Time Color */
```

Comment:
- [ ] Please move to `/* Override in special cases */`. (Line 169 and after)
```
case {{{GameplayPhase::C_Round}}}: {
	if (IsTimeLeft && RoundTimeLeft < 3000) LabelTimeLimit.TextPrefix = "$f00";
}
```

Reply:
sure.